test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -7276,6 +7276,146 @@ public void PackCommand_RequireLicenseAcceptanceNotEmitted()
                 }
             }
         }
+
+        [Fact]
+        public void PackCommand_NoRepo_GlobAllFiles_WithDefaultNuspec_Succeeds()
```

Comment:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming

The test should have two `_`'s. Is `NoRepo` a typo for `NoProjectFile`?

I think `PackCommand_NoProjectFileWithDefaultNuspec_GlobsAllFiles` meets our naming convention.